adss-rs/src/lib.rs

```diff
@@ -91,6 +100,10 @@ pub struct Commune {
   T: Option<Strobe>,
 }
 
+/// The `Share` struct holds the necessary data that is encoded in a
+/// single secret share. A share itself reveals nothing about the
+/// encoded secret data until it is paired with a threshold number of
+/// other shares.
```

Comment:
Above you define `AccessStructure` generically, so maybe, "... until it is combined with other shares according to the requirements defined by AccessStructure."